model_compression_toolkit/core/keras/pruning/check_node_role.py

```diff
@@ -0,0 +1,63 @@
+import keras.layers
```

Comment:
License, doc, return type...
(I'm not writing it for the rest of keras files... just go over all of them)